src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const sortedMovies = () => {
+    return moviesFromServer.filter(movie => {
+      const title = movie.title.toLowerCase();
+      const description = movie.description.toLowerCase();
+
+      return title.includes(query) || description.includes(query);
+    });
+  };
+
+  const searchedValue = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value.toLocaleLowerCase().trim());
```

Comment:
The trimming of input value may lead to unexpected results. Consider removing the trim method.